Link contributing guide to Element Web (#9220)
matrix-org/matrix-react-sdk: README.md

**Please file PRs against `develop`!!**

> Please follow the standard Matrix contributor's guide:

Reviewer: ```suggestion
We use the same contribution guide as Element. Check it out here:
```

https://github.com/vector-im/element-web/blob/develop/CONTRIBUTING.md

> Please follow the Matrix JS/React code style as per:

Reviewer: ```suggestion
Our code style is also the same as Element's:
```

> https://github.com/vector-im/element-web/blob/develop/code_style.md

Author: See https://github.com/vector-im/element-web/pull/23153
